otterdog/eclipse-pass.jsonnet

```diff
@@ -17,7 +17,18 @@ orgs.newOrg('eclipse-pass') {
       default_workflow_permissions: "write",
     },
   },
+  credentials+: [{
+      "provider": "bitwarden",
+      "item_id" : "23801ca4-fd27-446c-b5af-b07b0108f443"
+    },
+  ],
   secrets+: [
+    orgs.newOrgSecret('HELLO_WORLD_QUEST') {
```

Comment:
this is the correct way to define secrets.

The values are stored in credential providers maintained by the Eclipse Foundation. We currently use pass, and reference  them in a hierarchical structure, see the other secrets as an example. You again should not be concerned how the secret value is stored. You will need to provide it to us via a secure channel, and we add it to the provider and fill in the actual value as it is stored in the provider.

Reply:
@netomi how best to projects share these secrets with you?  Could we use the bitwarden account with a particular naming convention, or can we locally use `pass` and push those encrypted values into this repo?

Reply:
they are usually sent by encrypted email as far as I know, but can you chime in here @pstankie ?

Reply:
Yes, the secret can be sent via pgp encrypted email (https://gitlab.eclipse.org/pstankie.gpg) or via bitwarden send feature (https://bitwarden.com/help/about-send/)

Reply:
@pstankie do workgroups get access to EFs Bitwarden account (I have an account with my @ecelipse-foundation.org email, but I am wondering about the other team members).

Or, do workgroups need to manage their own Bitwarden accounts?

Here, for example, is an organization I created to (possibly) hold openaccess-pass secrets.
https://vault.bitwarden.com/#/vault?organizationId=e429b264-5b49-4794-be50-b0660125456a

Reply:
@pstankie or @netomi let me know if you have any documentation on how other workgroups manage their Bitwarden accounts (see comments above).

Thanks.

Reply:
Here's an example (password protected URL)
https://send.bitwarden.com/#R9KxxMqJiESP87ClATIJ-g/7_fOjgbzNTDWzyJqALdy_A

I can separately share that password offline / direct and then going forward we would keep using that shared secret so that we can post these share links in the repo.

Reply:
can you send the password via a gpg encrypted mail? My gpg key is available here: https://keyserver.ubuntu.com/pks/lookup?search=thomas.neidhart%40eclipse-foundation.org&fingerprint=on&op=index

Reply:
We have created a Wiki page at https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/wikis/Sharing-secrets to outline ways to share secrets with EF staff.

If there is no further need for adding a secret, please close this issue.